rclcpp_components/include/rclcpp_components/component_manager.hpp

```diff
@@ -131,6 +131,15 @@ class ComponentManager : public rclcpp::Node
   create_component_factory(const ComponentResource & resource);
 
 protected:
+  /// Create node options for loaded component
+  /*
+   * \param request information with the node to load
+   * \return node options
+   */
+  RCLCPP_COMPONENTS_PUBLIC
+  virtual rclcpp::NodeOptions
+  CreateNodeOptions(const std::shared_ptr<LoadNode::Request> request);
```

Comment:
I missed this when skimming before, but can we please change this to be `create_node_options`? We don't use CamelCase for methods in core ROS 2 packages like this one.

Reply:
Well, I just saw that more of these methods (protected ones only) are CamelCase... so I guess this can be left as-is for now... we really need to change them though.